src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -327,19 +333,45 @@ private ThreeState AssemblyDelaySignAttributeSetting
             }
         }
 
+        private static readonly Func<CommonAssemblyWellKnownAttributeData, string> s_assemblyKeyContainerGetter =
```

Comment:
>s_assemblyKeyContainerGetter [](http://example.com/codeflow?start=83&length=28)

Not sure what was the motivation to extract this lambda. It looks like it is used in one place and was likely cached by compiler anyway.